src/com/company/Main.java

```diff
@@ -2,7 +2,27 @@
 
 public class Main {
 
-    public static void main(String[] args) {
-	// write your code here
+    public static void main(String[] args){
+        int shift = 5;
+
+        String message = "uzgqnh%xyfynh%{tni%rfns-Xywnsl`b%fwlx.";
+        // System.out.println(Main.encrypt(shift, message));
+        System.out.println(Main.decrypt(shift, message));
+    }
+
+    public static String encrypt(int shift, String message){
```

Comment:
you can think how you can improve these two methods. they are almost identical

Reply:
Removed encrypt/decrypt methods and combined it into one - convertString() with adding boolean into signature.